include/circt/Dialect/SV/SVStatements.td

```diff
@@ -60,22 +60,34 @@ def IfOp : SVOp<"if", [HasRegionTerminator, NoRegionArguments,
                        RecursiveSideEffects]> {
   let summary = "'if (cond)' block";
   
-  let regions = (region SizedRegion<1>:$body);
+  let regions = (region SizedRegion<1>:$thenRegion, AnyRegion:$elseRegion);
```

Comment:
Should `$elseRegion` be a `SizedRegion<1>` instead of `AnyRegion`? Seems like the intent is for it to always have a [single block](https://github.com/llvm/circt/pull/415/files#diff-33cb6ea44b4d38d699fad5e0cbff3f382662456db511ee17b6fa12d536c1d3c5R89).

Reply:
It has a single block when present, but is empty when not.  This is how things like `affine.if` and `sv.ifdef` are modeled.

Reply:
Got it. I thought there was a different way to have something like an optional region, but I guess this is it.

Reply:
Yeah I can't say I find this very satisfactory either.  An `Optional<SizedRegion<1>>` seems much better to me.  If you're interested, you could ask on the mlir forums whether this has been considered and what it would take to get it into ODS.